### PR TITLE
Add community edition license for Teleport 16+

### PR DESCRIPTION
**Pull request by zmb3**
See https://goteleport.com/blog/teleport-community-license/ for details of this announcement.

### CODE REVIEW

File: LICENSE-community
Comment (zmb3) [inline LICENSE-community:1]: FYI @fheinecke @camscale this is the file that we should distribute in the tarballs and deb/rpm packages for OSS starting with Teleport 16.

**Review by rosstimothy — APPROVED** [review]

**Comment by fheinecke** [comment]
Shouldn't this have a changelog entry for v16? This is a pretty major change. Are we going to include notice of this with the v16 release in some other way?

**Comment by zmb3** [comment]
> Shouldn't this have a changelog entry for v16? This is a pretty major change. Are we going to include notice of this with the v16 release in some other way?

Yes, this is not the kind of item I want to be included as a small bullet-point in an auto-generated changelog. For major releases Roman I collaborate on a Google doc with complete release notes.

(Additionally, the UI in Teleport 16 will make it impossible for users to miss this change)

**Review by fheinecke — APPROVED** [review]
Thanks - just wanted to make sure that customers are notified.

File: LICENSE-community
Comment (camscale) [reply LICENSE-community:1]: Is there a license file we'd want to include in the enterprise artifacts for symmetry, or do we just leave it out?

I don't think we include a license file at all in any of the release artifacts. Should we add this license to more than just tarballs and deb/rpm packages? MacOS packages? OCI images, AMIs, whatever we do for WIndows?

Comment (fheinecke) [reply LICENSE-community:1]: > I don't think we include a license file at all in any of the release artifacts. 

We definitely do, in tarballs at least. We also have the `license` field of debs (and maybe rpms) set.

**Pull request merged**